pom.xml

```diff
@@ -30,29 +30,27 @@
 
   <properties>
     <apiary.receiver.sqs.verison>2.0.0</apiary.receiver.sqs.verison>
-    <cobertura.version>2.0.3</cobertura.version>
-    <cobertura.maven.plugin.version>2.6</cobertura.maven.plugin.version>
     <jdk.version>1.8</jdk.version>
     <slf4j.version>1.7.7</slf4j.version>
     <hadoop.version>2.7.2</hadoop.version>
     <hive.version>2.3.7</hive.version>
-    <hcommon.hive.metastore.version>1.4.1</hcommon.hive.metastore.version>
+    <hcommon.hive.metastore.version>1.4.2</hcommon.hive.metastore.version>
     <derby.version>10.10.2.0</derby.version>
     <kafka.version>0.11.0.1</kafka.version>
-    <aws.version>1.11.415</aws.version>
+    <aws.version>1.11.852</aws.version>
     <aws.kcl.version>1.9.0</aws.kcl.version>
     <aws.kpl.version>0.12.8</aws.kpl.version>
-    <circus.train.version>15.0.0</circus.train.version>
+    <circus.train.version>16.3.0</circus.train.version>
```

Comment:
this is a major version this won't affect SY? @abhimanyugupta07

Reply:
I looked through the release notes, the major version change in CT was due to Java 7 to 8 but SY is already Java 8 so that's fine. I don't think the rest has a direct impact on SY but as always appreciate another set of eyes. Everything compiles which is good ;)

Reply:
Yeah. I don't think it will break anything. 🤞

Reply:
I'll merge now.